src/DynamoRevit/DynamoRevit.cs

```diff
@@ -1100,9 +1100,7 @@ private static void OnSplashScreenClosed(object sender, EventArgs e)
             view.Closed -= OnSplashScreenClosed;
             //if the user explicitly closed the splashscreen, then we should let them boot
             //dynamo back up.
-            //TODO use new api by updating dynamocorewpf nuget.
-            //After it's published, remove the use of dynamic.
-            if(sender is Dynamo.UI.Views.SplashScreen ss && ((dynamic)ss).CloseWasExplicit)
+            if(sender is Dynamo.UI.Views.SplashScreen ss && ss.CloseWasExplicit)
```

Comment:
Do we need these changes in Revit 2024.2 as well?

Reply:
yes, or users will be able to open multiple instances of Dynamo.